[WIP] Update pd-overview-planning.md (#3100)
alan-turing-institute/the-turing-way: book/website/project-design/pd-overview/pd-overview-planning.md

### Operations 
* Confirm the budget and any funding policies you need to follow.
* Establish ways of working and collaborative practices for the project team.
* Identify roles and responsibilities within the project team, using the RACI matrix.
* Complete any institutional processes for project setup, such as ethics approval or contract signing.
* Useful documentation: [RACI matrix](https://project-management.com/understanding-responsibility-assignment-matrix-raci-matrix/), [risk register](https://asana.com/resources/risk-register), [project charter](https://www.projectmanager.com/blog/project-charter).

### Stakeholders
* Identify all individuals, groups, or organizations that have an interest or influence in the project. This includes both internal and external stakeholders
* Create a visual representation or matrix to understand the relationships between stakeholders and the project. Map their level of engagement, influence, and interest at each stage of the project. This helps prioritise engagement efforts and tailor communication strategies accordingly.

> * Utilise the "Facilitating Stakeholder Engagement" chapter for guidance and template resources [The chapter is being developed by RAM team #3105].

Author: ```suggestion
* Utilise the "Facilitating Stakeholder Engagement" chapter for guidance and template resources.
```

We can do a quick update once it's merged instead ;)

* Useful documentation: stakeholder register, stakeholder mapping, stakeholder analysis matrix, [personas](pd-persona).

### Outputs 
* Plan for the different outputs of your research, such as publications, software, or datasets.
* Consider licensing and copyright issues for sharing your outputs.
* Determine how you will manage intellectual property and ownership rights using an IP register.
* Remember to include any required reporting to funders.

> * Useful documentation: IP register, licensing and copyright statement, data sharing plan.

Author: Should just be data management plan again no? Also we should include 
```suggestion
* Useful documentation: [The Turing Way Guide to Communication](cm-comms-overview) IP register, [licensing](rr-licensing) and copyright statement, [data management plan](rr-rdm-dmp).
```

### Community & Communications 
* Identify the target audience for your research.
* Plan for effective communication and engagement with the community.
* Consider open and inclusive practices to involve stakeholders in decision-making.

> * Useful documentation: [Code of Conduct](ch-coc), [Contributing Guidelines](cl), [READMEs](https://the-turing-way.netlify.app/collaboration/github-novice/github-novice-firststeps.html?highlight=readmes), communication plan, stakeholder mapping, governance documentation.

Author: ```suggestion
* Useful documentation: [Code of Conduct](ch-coc), [Contributing Guidelines](cl), [READMEs](https://the-turing-way.netlify.app/collaboration/github-novice/github-novice-firststeps.html?highlight=readmes), communication plan, stakeholder mapping, [governance documentation](er-ethics-open-source-governance).
```

* Complete any institutional processes for project set up, such as ethics approval or contracting signing. 
* Useful documentation: RACI matrix, risk register, project charter.

### Stakeholders
* Identify the project stakeholders
* Complete a stakeholder mapping, identify level of engagement and stakeholder responsibilities on each stage of the  project
* Use the facilitating stakeholder engagement for guidance and templates resources[quote the chapter being developed by RAM team #3105]

> 

Reviewer: I added a few documentation/tools!
